easybuild/easyconfigs/c/CellProfiler/CellProfiler-4.2.8-foss-2023a.eb

```
@@ -0,0 +1,126 @@
+# Based on EasyConfig from Jörg Saßmannshausen's PR:
+# https://github.com/easybuilders/easybuild-easyconfigs/pull/20725
+
+
+easyblock = 'PythonBundle'
+
+name = 'CellProfiler'
+version = '4.2.8'
+
+homepage = 'http://cellprofiler.org/'
+description = """CellProfiler is a free open-source software designed to enable 
+biologists without training in computer vision or programming to quantitatively 
+measure phenotypes from thousands of images automatically."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+builddependencies = [
+    ('pkgconf', '1.9.5'),
+    ('hypothesis', '6.82.0'),
+    ('pkgconfig', '1.5.5', '-python'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('Python-bundle-PyPI', '2023.06'),
+    ('matplotlib', '3.7.2'),
+    ('Java', '11', '', True),
```

use `SYSTEM` constant (to fix CI):
```suggestion
    ('Java', '11', '', SYSTEM),
```